pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -505,12 +505,16 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
 	statuses := make([]*kubecontainer.Status, len(containers))
 	// TODO: optimization: set maximum number of containers per container name to examine.
 	for i, c := range containers {
-		status, err := m.runtimeService.ContainerStatus(c.Id)
+		resp, err := m.runtimeService.ContainerStatus(c.Id, false)
 		if err != nil {
 			// Merely log this here; GetPodStatus will actually report the error out.
 			klog.V(4).InfoS("ContainerStatus return error", "containerID", c.Id, "err", err)
 			return nil, err
 		}
+		status := resp.GetStatus()
+		if status == nil {
+			return nil, errors.New("container status is nil")
+		}
```

Comment:
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/errors.go#L28 

Nit: a constant like [ErrNetworkUnknown](https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/errors.go#L28 ) may be better

Reply:
Fixed, added a new error to the `remote` package.